paddle/fluid/operators/mkldnn/fc_mkldnn_op.cc

```diff
@@ -37,7 +37,7 @@ using mkldnn::primitive;
 using mkldnn::stream;
```

Comment:
This `fc` op is too complex to understand. Please give more comments.

Reply:
This PR introduces changes on 3 levels:
1. Pass that enables FC (`fc_mkldnn_pass`)
![Pass](https://user-images.githubusercontent.com/4967505/65517962-d5f42300-dee3-11e9-9957-0b58eb145eee.png)
The requirement in the last step is required so that int8 kernel can work properly on one of the FC layers found in VGG16 and VGG19 network.

2. MKL-DNN Quantizer, so that FC-related tensors can have their scales computed.
![Quantizer](https://user-images.githubusercontent.com/4967505/65517968-d987aa00-dee3-11e9-8632-858aedca6522.png)

3. `fc_mkldnn_op` - so that the int8 part can be incorporated.
The control flow when instruction pointer enters MKL-DNN's FC op looks as follows:
![FC Kernel](https://user-images.githubusercontent.com/4967505/65517978-df7d8b00-dee3-11e9-8691-924f79f81916.png)